pkg/util/provider/machinecontroller/machine.go

```diff
@@ -433,6 +433,7 @@ func (c *controller) triggerCreationFlow(ctx context.Context, createMachineReque
 				}
 			} else {
 				//if node label present that means there must be a backing VM ,without need of GetMachineStatus() call
+				klog.V(2).Infof("VM for machine %q should be visible in some time", machine.Name)
```

Comment:
Do you think this log is really needed?

Reply:
I added it to catch the delay in the machine visibility . Sure I think it can be removed.